trieste/bayesian_optimizer.py

```diff
@@ -312,29 +343,33 @@ def optimize(
                     f" {OBJECTIVE!r}, got keys {datasets.keys()}"
                 )
 
-            acquisition_rule = cast(AcquisitionRule[S, SP], EfficientGlobalOptimization())
+            acquisition_rule = cast(AcquisitionRule[SP], EfficientGlobalOptimization())
 
         models = map_values(create_model, model_specs)
         history: list[Record[S]] = []
 
         for step in range(num_steps):
             if track_state:
-                history.append(Record(datasets, models, acquisition_state))
+                history.append(Record(datasets, models, trust_region_state))
 
             try:
                 if track_state:
                     models = copy.deepcopy(models)
-                    acquisition_state = copy.deepcopy(acquisition_state)
+                    trust_region_state = copy.deepcopy(trust_region_state)
 
-                if step == 0 and fit_intial_model:
+                if step == 0 and fit_initial_model:
                     for tag, model in models.items():
                         dataset = datasets[tag]
                         model.update(dataset)
                         model.optimize(dataset)
 
-                query_points, acquisition_state = acquisition_rule.acquire(
-                    self._search_space, datasets, models, acquisition_state
-                )
+                if trust_region is None:
+                    acquisition_space = self._search_space
```

Comment:
we could make a "unit" empiric and use that, but I felt that was unnecessary for this one use case